packages/components/src/flex/flex/hook.ts

```diff
@@ -22,7 +22,7 @@ import type { FlexProps } from '../types';
 
 function useDeprecatedProps(
 	props: WordPressComponentProps< FlexProps, 'div' >
-): WordPressComponentProps< FlexProps, 'div' > {
+): Omit< typeof props, 'isReversed' > {
```

Comment:
This return type was wrong because it wasn't omitting the deprecated `isReversed` prop, which is specifically being omitted in this hook. It wasn't causing a TS error per se, but it was making it look like `useFlex` could return another invalid prop (`isReversed`) in addition to `isColumn`.